tests/test_api.py

```diff
@@ -1395,3 +1395,36 @@ def test_additional_sideloads(self):
         self.assertTrue('+cats' in content)
         self.assertEquals(content['cat']['name'], 'Kitten')
         self.assertEquals(content['+cats'][0]['name'], 'Parent')
+
+    def test_immutable_field(self):
+        """ Make sure immutable 'parent' field can be set on POST """
+        parent_id = self.kitten.parent_id
+        data = {
+            'name': 'New Kitten',
+            'home': self.kitten.home_id,
+            'backup_home': self.kitten.backup_home_id,
+            'parent': parent_id
+        }
+        response = self.client.post(
+            '/cats/',
+            json.dumps(data),
+            content_type='application/json'
+        )
+        self.assertEqual(201, response.status_code)
+        data = json.loads(response.content.decode('utf-8'))
+        self.assertEqual(data['cat']['parent'], parent_id)
+
+        # Try to change 'parent' in a PATCH request...
+        patch_data = {
+            'parent': self.kitten.pk
```

Comment:
Should we add `immutable` to our `OPTIONS` response? Currently, the behavior for read-only fields is to ignore any new values that might be passed by the client, which is fine... but at least read-only fields can be identified as such by the client using OPTIONS. With immutable fields, there is currently no way of knowing.